package.json

```diff
@@ -4,11 +4,13 @@
   "description": "Arbitration Outsourcing for State Channels",
   "main": "index.js",
   "scripts": {
-    "test": "npx tsc && mocha build/__pisa_tests__ --exit",
+    "test": "npm run build && mocha $(find ./build/__pisa_tests__/*.js ! -path '*quickTest*') --exit",
```

Comment:
I dont understand why this needs to be done at all? wouldnt the command as it was work? it was execeuting all tests within __pisa_tests__ directory

Reply:
yeah, I didn't want it to execute the quick tests. These wont work when executed on circleci. I could just move them to another folder within source?

Reply:
yes please, it will be simpler that way

Reply:
cool, done